synfig-studio/src/gui/app.cpp

```diff
@@ -1057,13 +1057,32 @@ DEFINE_ACTION("help-about",     Gtk::StockID("synfig-about"))
 // actions: Keyframe
 DEFINE_ACTION("keyframe-properties", _("Properties"))
 
+// actions: move to tab
+DEFINE_ACTION("move-to-tab-1",  _("synfig-move-to-tab-1"))
+DEFINE_ACTION("move-to-tab-2",  _("synfig-move-to-tab-2"))
+DEFINE_ACTION("move-to-tab-3",  _("synfig-move-to-tab-3"))
+DEFINE_ACTION("move-to-tab-4",  _("synfig-move-to-tab-4"))
+DEFINE_ACTION("move-to-tab-5",  _("synfig-move-to-tab-5"))
+DEFINE_ACTION("move-to-tab-6",  _("synfig-move-to-tab-6"))
+DEFINE_ACTION("move-to-tab-7",  _("synfig-move-to-tab-7"))
+DEFINE_ACTION("move-to-tab-8",  _("synfig-move-to-tab-8"))
+DEFINE_ACTION("move-to-tab-last",  _("synfig-move-to-tab-last"))
 
 //Layout the actions in the main menu (caret menu, right click on canvas menu) and toolbar:
 	Glib::ustring ui_info_menu =
 "	<menu action='menu-file'>"
 "		<menuitem action='new' />"
 "		<menuitem action='open' />"
 "		<menu action='menu-open-recent' />"
+"		<menu action='move-to-tab-1' />"
```

Comment:
Can you explain why do you add this actions to the menu?

Reply:
before adding this the action wasn't working. i.e the callback wasn't being called so I read that for an action to function properly it must be added as a menu item even if it doesn't show and sure enough that did the trick.

Reply:
There is a "hidden-menu" somewhere else, to avoid showing stuff in the menu.

Reply:
hmm, ok seems better to move this there then.

Reply:
done, I guess this can now be safely resolved :)